test/unittests/test_analyse_xml_data/test_construct_and_analyze_graph.py

```diff
@@ -117,8 +117,8 @@ def test_extract_id_one_text_with_several_id() -> None:
     test_ele = etree.fromstring(
         """<text permissions="prop-default" encoding="xml">
             <a class="salsah-link" href="IRI:res_A_11:IRI">res_A_11</a>
-            <a class="salsah-link" href="IRI:res_B_11:IRI">res_A_11</a>
-            <a class="salsah-link" href="IRI:res_B_11:IRI">res_A_11</a>
+            <a class="salsah-link" href="IRI:res_B_11:IRI">res_B_11</a>
+            <a class="salsah-link" href="IRI:res_B_11:IRI">res_B_11</a>
         </text>"""
```

Comment:
I'm not sure I understand 100%... is this a typo you're fixing? or is it a different scenario? if the latter is the case, I would propose not changing the existing test, but to make this an additional test case. In the fist case, this makes sense, of course

Reply:
just a typo